LiskHQ/lisk-hub
# Add meta tags - Closes #702

### What was the problem?
We had no og meta tags (like description)

### How did I fix it?
 By adding them
- keywords
- title
- url
- description
- image (will be visible once live)

### How to test it?
send the jenkins build to yourself on e.g. slack, it should look like this:
<img width="558" alt="screen shot 2018-04-10 at 14 46 59" src="https://user-images.githubusercontent.com/9592216/38557756-53018bd0-3cce-11e8-86cd-0a73fb2bc8c6.png">

### Review checklist
- The PR solves #702
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
